dump_frames.py

```diff
@@ -128,25 +125,26 @@ def main():
     parser.add_argument('output_directory',
                         default=None,
                         help='Directory to output frames to.')
-    parser.add_argument('--frames_per_second',
-                        default=1,
-                        help=('Number of frames to output per second. If 0, '
-                            'dumps all frames in the clip'))
-    parser.add_argument('--num-workers', type=int, default=4)
+    parser.add_argument('--fps',
+                        default=None,
+                        help=('Number of frames to output per second. If not '
+                             'specified, dumps all frames in the clip.'))
```

Comment:
Nit: Can you leave the special casing for 0? I believe you can't specify "None" from the command line / shell scripts, so the only way to dump all frames is to not specify --fps at all, which leads to some annoyances in shell scripts. E.g. things like
```
if [[ ${DUMP_ALL_FRAMES} ]] ; then
py dump_frames.py ... --fps ${FPS} 
else
py dump_frames.py ... --fps ${FPS} 
fi
```